src/js/vic-v2/config/form.js

```diff
@@ -23,6 +24,7 @@ const {
   email,
   serviceBranch,
   dd214
+  // photo
```

Comment:
The photo schema needs to be updated to reflect the structure provided below. Should a file definition be created, and referenced by the photo schema?

Reply:
There should be a file definition already (or we need to pull it out of the multiple files definition), we should update the schema to reference that, yeah.